src/libi_sharing/models.py

```diff
@@ -1,3 +1,63 @@
 from django.db import models
 
-# Create your models here.
+from libi_common.models import BaseModel
+
+
+class Sharing(BaseModel):
+    title = models.CharField(blank=False, max_length=200)
+    description = models.TextField(blank=True)
+    section = models.ForeignKey('Section', null=True, on_delete=models.SET_NULL)
+    category = models.ForeignKey('Category', null=True, on_delete=models.SET_NULL)
+    created_account = models.ForeignKey('libi_account.Account', null=True, on_delete=models.SET_NULL)
+    price = models.IntegerField()
+    sharing_type = models.IntegerField(blank=False, null=False)
```

Comment:
```suggestion
    sharing_type = models.IntegerField(null=False)
```
https://docs.djangoproject.com/en/3.0/ref/models/fields/#blank
`blank` option은 CharField, TextField 등에서 유효한 옵션으로 보이며, IntegerField에서는 제거하는게 어떨까요?